examples/tfengine/team.hcl

```diff
@@ -239,7 +239,6 @@ template "project_data" {
           type               = "mysql"
           network_project_id = "example-prod-networks"
           network            = "example-network"
-          subnet             = "example-subnet"
```

Comment:
Is subnet not used for Cloud SQL instances?

Reply:
it's not